src/App.jsx

```diff
@@ -14,57 +16,57 @@ export const goods = [
   'Garlic',
 ];
 
```

Comment:
I recommend using separate functions for cleaning and product selection, making the code more readable and understandable.
Example:
```
const handleSelect = (good) => {};
```
```
const handleClear = () => {};
```